lib/browser/api/menu.ts

```diff
@@ -78,6 +78,7 @@ Menu.prototype.popup = function (options = {}) {
   if (typeof x !== 'number') x = -1;
   if (typeof y !== 'number') y = -1;
   if (typeof positioningItem !== 'number') positioningItem = -1;
+  if (typeof sourceType !== 'string' || !sourceType) sourceType = 'mouse';
```

Comment:
Why default to `mouse`? Why not `none`?

Reply:
mouse was the old default, and is also the default upstream iirc.  Don't want to go breaking anything 😅